### PR TITLE
Update Chinese translation

### PR DESCRIPTION
**Pull request by Vstory**

### CODE REVIEW

**Comment by Vstory** [comment]
> Thanks a lot for updating the Chinese translation! Did you miss some sentences down at the bottom there?

I deliberately did not translate it, keeping it consistent with the Chinese translation in version 1.x.


> On a side note, please push any changes you make after creating a pull request to the same pull request, instead of closing it and creating a new one.

I know that closing the issue is not correct, and I should put the modified content in the same issue.
I use the github.com web page instead of GitHub Desktop.I cannot load the Aegis project on the GitHub desktop due to GFW issues. I have already used a VPN, but I don’t know what the reason is.

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 1]
```diff
@@ -4,34 +4,46 @@
     <string name="action_about">关于</string>
     <string name="action_import">导入</string>
     <string name="action_delete">删除</string>
+    <string name="action_edit_icon">编辑图标</string>
     <string name="action_default_icon">恢复默认图标</string>
     <string name="discard">放弃</string>
     <string name="save">保存</string>
-    <string name="issuer">服务商</string>
+    <string name="issuer">Issuer</string>
```
[the remark this answers is not in the document]
Comment (Vstory) [reply app/src/main/res/values-zh-rCN/strings.xml:11]: Missing.

**Comment by alexbakker** [comment]
Thanks! Please squash all of your changes into a single commit.

**Comment by Vstory** [comment]
> Thanks! Please squash all of your changes into a single commit.

Sorry, I am not familiar with github.  I don't understand what you mean.
[Last version](https://github.com/beemdevelopment/Aegis/pull/470/commits/472aa05ad150b7810d2e5f02d66b83d218d105b4)

**Comment by alexbakker** [comment]
That's alright. We try to keep a clean commit history. Your branch had 8 commits and I was asking if you could rewrite the history so that there's only 1 commit for your changes (https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#_squashing).

I've squashed the commits for you and pushed it to your branch.

Thanks again!

**Pull request merged**